be/src/storage/lake/lake_primary_index.h

```diff
@@ -33,7 +33,7 @@ class LakePrimaryIndex : public PrimaryIndex {
 public:
     LakePrimaryIndex() : PrimaryIndex() {}
     LakePrimaryIndex(const Schema& pk_schema) : PrimaryIndex(pk_schema) {}
-    ~LakePrimaryIndex() = default;
+    ~LakePrimaryIndex();
```

Comment:
```suggestion
    ~LakePrimaryIndex() override;
```
and make the destructor of `PrimaryIndex` a virtual method.